components/alibi-detect-server/Dockerfile

```diff
@@ -59,12 +69,6 @@ COPY version.txt version.txt
 FROM base as final
 WORKDIR /microservice
 
-# this is to avoid "ImportError: libGL.so.1" from opencv
-RUN yum install -y mesa-libGL
-RUN yum -y update-minimal --security --sec-severity=Important --sec-severity=Critical
```

Comment:
Are we sure we wanna skip the line with security updates?

Reply:
Good spot!

I moved them all together to the top of the `Dockerfile`(s), and changed them to `microdnf update -y` (which is the closest equivalent for `microdnf`). 

In some cases, like the one you mentioned, I added it to the interim multi-stage builds, but forgot to add it to the final one as well. I'll double check them to ensure the `microdnf update -y` is present.